nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/java/org/nuxeo/ecm/platform/audit/service/LogEntryProvider.java

```diff
@@ -89,7 +89,11 @@ protected void doPersist(LogEntry entry) {
         Object entry = entries.get(0);
         if (entry instanceof LogEntry) {
             for (Object logEntry : entries) {
-                doPublish((LogEntry) logEntry);
+                try {
+                    doPublish((LogEntry) logEntry);
+                } catch (RuntimeException e) {
```

Comment:
Which exception has been observed here? Please don't catch exceptions when we don't actually know what they are.